DetectorDescription/DDCMS/plugins/dd4hep/DDDefinitions2Objects.cc

```diff
@@ -1840,6 +1849,47 @@ static long load_dddefinition(Detector& det, xml_h element) {
       print_doc((doc = d).root());
       xml_coll_t(d.root(), DD_CMU(MaterialSection)).for_each(Converter<MaterialSection>(det, &context));
     }
+    {
+      printout(context.debug_materials ? ALWAYS : DEBUG,
+               "DD4CMS",
```

Comment:
@ianna question: is this DD4hep function thread-safe? In the same spirit as we prefer MessageLogger to std::cout

Reply:
yes, it is thread-safe.